src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -139,7 +139,7 @@ functionsToRunOnDataTablesDrawEvent: [],
         localStorage.setItem('{{ Str::slug($crud->getRoute()) }}_list_url', newUrl);
       },
       dataTableConfiguration: {
```

Comment:
Question: what happens for someone who's published their `datatables_logic.blade.php` file?
- they do `composer update`
- they get the new PHP files
- they do NOT get this new blade file

In this case... does the list operation break for them?

Reply:
Very interesting question, I haven't thought about it. 

I've just tested it, pulled this branch, and replaced this branch `datatables_logic` blade file with the current version in crud.

Everything works normally unless developer tries to set `showEntryCount => false`. In that case the pagination and the entry count get's messy. 

This branch in it's current state: 
![image](https://user-images.githubusercontent.com/7188159/193799920-5bf0418c-9c90-4081-a147-f62e55884544.png)

This branch with datatables logic replaced:
![image](https://user-images.githubusercontent.com/7188159/193799721-b364dd5f-f488-42d3-b397-c08205764a11.png)

To sum up, any project will work as before ... but if they try to set `showEntryCount` without updating `datatables_logic` it will fail. 
Could this be solved by documentation ? I don't think we can consider this a BC if it does not break the "default" behaviour, and we are not forcing it, developer need to opt-in for `showEntryCount => false`.

Let me know what you think.

Cheers

Reply:
EXCELLEEEEENT! We can consider it a non-breaking change then. Yeah, we can easily solve this with documentation - where we document `showEntryCount` we just tell people to double-check the blade view isn't published 🎉